src/features/defi/components/GetStarted/GetStarted.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, VStack } from '@chakra-ui/react'
+import osmosis from 'assets/osmosis.svg'
+import { Text } from 'components/Text'
+
+import { DefiModalHeader } from '../DefiModalHeader/DefiModalHeader'
+
+type GetStartedProps = {
+  assetId: string
+}
+
+const ASSET_ID_TO_MAX_APR = {
+  'cosmoshub-3/slip44:118': '12'
+}
+
+export const GetStarted = ({ assetId }: GetStartedProps) => {
+  // TODO: parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmo'
+  }))(assetId)
```

Comment:
Didn't you created a selector for this? 🤣

Reply:
Yup, but it still needs to be wired up

Reply:
mb